### PR TITLE
feat: enable CCTP in mainnet and misc fixes

### PR DESCRIPTION
**Pull request by 0xArdy**
closes SUP-97

### CODE REVIEW

File: src/components/TokenButton.tsx [hunk 1]
```diff
@@ -12,8 +12,18 @@ interface TokenButtonProps {
...
-    <StyledButton onClick={onClick} endIcon={<Image src={chevronDown} alt='chevrown down' width={16} height={16} />}>
-      <Image src={selectedToken?.logoURI} alt={selectedToken?.name} className='token-image' width={32} height={32} />
+    <StyledButton
+      onClick={onClick}
+      disabled={!selectedToken?.address}
+      endIcon={<Image src={chevronDown} alt='chevrown down' width={16} height={16} />}
+    >
+      <Image
+        src={selectedToken?.logoURI || ''}
+        alt={selectedToken?.name || ''}
```
Comment (turtlemoji) [inline src/components/TokenButton.tsx:22]:
```suggestion
        alt={selectedToken?.name || 'Token Logo'}
```

File: src/data/cctp.json [hunk 2]
```diff
@@ -1,4 +1,22 @@
 {
+  "1": {
+    "domain": 0,
+    "tokenMessenger": "0xbd3fa81b58ba92a82136038b25adec7066af3155",
```
Comment (turtlemoji) [inline src/data/cctp.json:4]: triple check

File: src/data/extraTokens.json [hunk 3]
```diff
@@ -1,4 +1,31 @@
 [
+  {
+    "chainId": 1,
+    "address": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
```
Comment (turtlemoji) [inline src/data/extraTokens.json:4]: triple check

**Review by turtlemoji — APPROVED** [review]

**Review by 0xtiti — APPROVED** [review]

File: src/data/extraTokens.json (hunk 3 above)
Comment (0xArdy) [reply src/data/extraTokens.json:4]: done!

File: src/data/cctp.json (hunk 2 above)
Comment (0xArdy) [reply src/data/cctp.json:4]: done :heavy_check_mark:

**Pull request merged**